### PR TITLE
[CloudPC] Add cloudPcInGracePeriodScenario in public alert doc

### PR DESCRIPTION
**Pull request by zansy**
Add cloudPcInGracePeriodScenario in public alert doc
the CSDL and api.md files for the PR:
[Add in grace period scenario to alertRuleTemplate](https://microsoftgraph.visualstudio.com/onboarding/_git/onboarding/pullrequest/12969)

### CODE REVIEW

File: api-reference/beta/resources/devicemanagement-alertrule.md [hunk 1]
```diff
@@ -50,6 +50,7 @@ For more information, see the [monitoring](devicemanagement-monitoring.md) resou
 |cloudPcImageUploadScenario|Indicates that the alert rule was triggered for an issue with the process to upload the Cloud PC image. For a system rule, the alert rule was triggered for a failure to upload the Cloud PC image.|
 |cloudPcOnPremiseNetworkConnectionCheckScenario|Indicates that the alert rule was triggered for an issue with the on-premises network connection check. For a system rule, the alert rule was triggered for a failure with the on-premises network connection.|
 |unknownFutureValue|Evolvable enumeration sentinel value. Do not use.|
+|cloudPcInGracePeriodScenario |Indicates that the alert rule was triggered when Cloud PC entered the in grace period. For a system rule, the alert rule was triggered when Cloud PC entered the in grace period.|
```
Comment (JarbasHorst) [inline api-reference/beta/resources/devicemanagement-alertrule.md:53]: Hi @zansy, it appears that the same information is being repeated twice in both sentences. I am missing something or is the behavior the same for system rules and non-system rules? I recommend you to rephrase this description. Suggested revision:

- Indicates that the alert rule was triggered when the Cloud PC entered the grace period, regardless of whether it was a system rule or not.

File: api-reference/beta/api/devicemanagement-alertrule-post.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ POST /deviceManagement/monitoring/alertRules
 
 |Property|Type|Description|
 |:---|:---|:---|
-|alertRuleTemplate|microsoft.graph.deviceManagement.alertRuleTemplate|The rule template of the alert event. The possible values are: `cloudPcProvisionScenario`, `cloudPcImageUploadScenario`, `cloudPcOnPremiseNetworkConnectionCheckScenario`, `unknownFutureValue`.|
+|alertRuleTemplate|microsoft.graph.deviceManagement.alertRuleTemplate|The rule template of the alert event. The possible values are: `cloudPcProvisionScenario`, `cloudPcImageUploadScenario`, `cloudPcOnPremiseNetworkConnectionCheckScenario`, `unknownFutureValue`, `cloudPcInGracePeriodScenario`. Note that you must use the `Prefer: include-unknown-enum-members` request header to get the following values from this [evolvable enum](/graph/best-practices-concept#handling-future-members-in-evolvable-enumerations): `cloudPcInGracePeriodScenario`.|
```
Comment (JarbasHorst) [inline api-reference/beta/api/devicemanagement-alertrule-post.md:51]: I have also update the POST operation of the alertRule resource. Please, let me know if this is not correct.

**Comment by JarbasHorst** [comment]
Hello @zansy, I reviewed your PR and left some comments. Please, take a look when you will have time.

**Comment by JarbasHorst** [comment]
Hi @Lauragra, this PR is ready for final review.

**Review by Lauragra — APPROVED** [review]

**Review by RuiHou105 — APPROVED** [review]

**Comment by JarbasHorst** [comment]
Hi @zansy, there is nothing else needed in this PR from my side. Your PR has the status "content review complete". Please add the label "ready to merge" when you are ready to publish your PR. Thanks :)

**Pull request merged**